Jenkinsfile_CNP

```diff
@@ -83,7 +83,6 @@ withPipeline(type, product, component) {
   enableAksStagingDeployment()
   installCharts()
   enableSlackNotifications(channel)
-  enableApiGatewayTest()
```

Comment:
worth commenting it rather removing as platform will fix the bug and we might forget ?